.travis.yml

```diff
@@ -1,45 +1,53 @@
 language: node_js
-
+dist: trusty
 compiler: gcc
 sudo: false
 
 env:
   global:
+    # https://github.com/jasongin/nvs/blob/master/doc/CI.md
+    - NVS_VERSION=1.3.0
     - SKIP_SASS_BINARY_DOWNLOAD_FOR_CI=true
 
 jobs:
   include:
     - stage: test
-      node_js: "node"
+      NODEJS_VERSION: node/6
       os: linux
       before_script: npm run lint || exit 1;
       after_success: npm run-script coverage;
     - stage: platform-test
-      node_js: "node"
+      NODEJS_VERSION: node/8
       os: osx    
     - stage: platform-test
-      node_js: "7"
+      NODEJS_VERSION: node/7
       os: linux
     - stage: platform-test
-      node_js: "7"
+      NODEJS_VERSION: node/7
       os: osx
     - stage: platform-test
-      node_js: "lts/boron"
+      NODEJS_VERSION: node/boron
       os: linux
     - stage: platform-test
-      node_js: "lts/boron"
+      NODEJS_VERSION: node/boron
       os: osx
     - stage: platform-test
-      node_js: "lts/argon"
+      NODEJS_VERSION: node/argon
       os: linux
     - stage: platform-test
-      node_js: "lts/argon"
+      NODEJS_VERSION: node/argon
       os: osx
     - stage: platform-test
-      node_js: "0.12"
+      NODEJS_VERSION: node/0.12
+      os: linux
+    - stage: platform-test
+      NODEJS_VERSION: node/0.10
       os: linux
     - stage: platform-test
-      node_js: "0.10"
+      NODEJS_VERSION: chakracore/latest
```

Comment:
I reviewed that option, but the way travis works is if you specify `nvs_use`, it will internally use `nvm` to install node. There is no easy way to have both travis's `nvm` and `nvs` at same time. Is there any concern of using `nvs` that I can address in this PR?